website/docs/reference/supported_commands.md

```diff
@@ -135,6 +135,15 @@ Related epic - [Issue](https://github.com/FerretDB/FerretDB/issues/78).
 |                            | `indexes`                        | ⚠️      |                                                           |
 |                            | `comment`                        | ⚠️      |                                                           |
 
+## Free Monitoring Commands
+
+| Command                    | Argument                             | Status | Comments                                                                      |
+|----------------------------|--------------------------------------|--------|-------------------------------------------------------------------------------|
+| `setFreeMonitoring`        |                                      | ✅     | [Telemetry reporting](https://docs.ferretdb.io/telemetry/)                    |
+|                            | `action: "enable"`                   | ✅     | [`--telemetry=enable`](https://docs.ferretdb.io/telemetry/#enable-telemetry)  |
+|                            | `action: "disable"`                  | ✅     | [`--telemetry=disable`](https://docs.ferretdb.io/telemetry/#enable-telemetry) |
```

Comment:
Also I don't quite understand why do we mention `--telemetry` flag, which is a flag provided to FerretDB on startup. How is it related to the `setFreeMonitoring` command, which can be run on runtime and only if `--telemetry` flag is not provided. Am I missing something? :slightly_smiling_face:

Reply:
The `setFreeMonitoring` command is disabled, when a telemetry state was set via `--telemetry` flag / environment variable / etc...

So it can be actually used if telemetry is not set on startup

Regarding my comment though, now I think that informing about a flag here looks reasonable, because it links to the article where we inform about above rule :)